base/util.jl

```diff
@@ -577,7 +577,16 @@ function _kwdef!(blk, params_args, call_args)
             push!(params_args, ei)
             push!(call_args, ei)
         elseif ei isa Expr
-            if ei.head === :(=)
+            is_atomic = ei.head == :atomic
+            ei = is_atomic ? first(ei.args) : ei # strip "@atomic" and add it back later
+            is_const = ei.head == :const
```

Comment:
```suggestion
            is_atomic = ei.head === :atomic
            ei = is_atomic ? first(ei.args) : ei # strip "@atomic" and add it back later
            is_const = ei.head === :const
```
As a convention, we generally use `===` for `Symbol` comparison (at least within the Base`).

Reply:
With this fixed do you think this is good to go?